tests/test_base.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import, division, print_function
+import unittest
+
+import numpy as np
+from numpy.testing import assert_equal
+
+import astshim as ast
+from astshim.test import ObjectTestCase
+
+
+class TestObject(ObjectTestCase):
+
+    def test_arrayFromVector(self):
+        nAxes = 3
+        nValues = 5
+        dataVec = np.random.rand(nAxes * nValues)
+        desiredDataArr = dataVec.copy()
+        desiredDataArr.shape = (nAxes, nValues)
+        dataArr = ast.arrayFromVector(vec=dataVec, nAxes=nAxes)
```

Comment:
Isn't `dataVec` a `numpy.ndarray`? Would it be useful to test this with a `list`?

Reply:
I agree. I'll test both ways.